app_dart/lib/src/request_handlers/query_github_graphql.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:appengine/appengine.dart';
+import 'package:cocoon_service/cocoon_service.dart';
+import 'package:cocoon_service/src/request_handling/exceptions.dart';
+import 'package:graphql/client.dart';
+import 'package:meta/meta.dart';
+
+import '../datastore/config.dart';
+import '../request_handling/api_request_handler.dart';
+import '../request_handling/authentication.dart';
+import '../request_handling/body.dart';
+
+/// Runs all the applicable tasks for a given PR and commit hash. This will be
+/// used to unblock rollers when creating a new commit is not possible.
+@immutable
+class QueryGithubGraphql extends ApiRequestHandler<Body> {
+  const QueryGithubGraphql(
+    Config config,
+    AuthenticationProvider authenticationProvider,
+  ) : super(config: config, authenticationProvider: authenticationProvider);
+
+  @override
+  Future<Body> post() async {
+    final String requestDataString = String.fromCharCodes(requestBody);
+    log.error(requestDataString);
```

Comment:
should this be a lower log level (info, debug?)

Reply:
uppps wrong file

> How does this PR surface builds in MILO?



> How does this PR surface builds in MILO?

wrong file, uploaded the correct one.